powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java

```diff
@@ -195,9 +195,9 @@ private void processDocuments(final TestContext context) throws Exception {
         submitDocuments(idCardSubmitRequest, idCardSubmits);
 
         if (config.isVerificationOnSubmitEnabled()) {
-            assertStatusOfSubmittedDocsWithRetries(processId, idCardSubmits.size(), DocumentStatus.VERIFICATION_PENDING);
+            assertStatusOfSubmittedDocsWithRetries(processId, idCardSubmits.size(), DocumentStatus.ACCEPTED);
```

Comment:
`document/status` does not change status anymore, it is done by async task, so document may go far.